### PR TITLE
Pull Request for Issue1604: Add pile up peaks to AMSingleSpectrumView

### PR DESCRIPTION
**Pull request by dretrex**
I essentially ported over the existing code from AMXRFDetailedDetectorView to do this, therefore there isn't a lot of new code.  Just duplicated code.

### CODE REVIEW

File: source/ui/dataman/AM2DScanView.cpp [hunk 1]
```diff
@@ -492,7 +492,52 @@ void AM2DScanView::mousePressEvent(QMouseEvent *e)
 
 void AM2DScanView::setPlotRange(double low, double high)
 {
-	spectrumView_->setPlotRange(low, high);
```
Comment (helfrij) [inline source/ui/dataman/AM2DScanView.cpp]: Just curious, what's the difference between setPlotRange(...) and setEnergyRange(...) for the spectrum view? Or has this method just been renamed?

Comment (dretrex) [inline source/ui/dataman/AM2DScanView.cpp]: It has been renamed in the spectrum view.

Comment (helfrij) [inline source/ui/dataman/AM2DScanView.cpp]: Since the scan view methods just call this renamed function, should the scan view methods be renamed as well?

**Comment by helfrij** [comment]
Looks good!

Time Estimate: 15 minutes.

File: source/ui/dataman/AM2DScanView.cpp (hunk 1 above)
Comment (dretrex) [inline source/ui/dataman/AM2DScanView.cpp]: Yeah, that's probably a good idea.  I'll make that change.

**Pull request merged**